packages/core/src/builder.class.ts

```diff
@@ -1263,15 +1271,11 @@ export class Builder {
         const url = parse(event.origin);
         const isRestricted =
           ['builder.register', 'builder.registerComponent'].indexOf(event.data?.type) === -1;
-        if (
-          isRestricted &&
-          !(
-            url.hostname &&
-            (url.hostname === 'builder.io' ||
-              url.hostname.endsWith('.builder.io') ||
```

Comment:
this update negates this piece of code - we want to support **.builder.io, not just builder.io

Reply:
updated to use `isTrusted(host)` which only check if the `builder.io` is in the host string

Reply:
👍